packages/protocol/contracts/identity/Attestations.sol

```diff
@@ -141,17 +98,78 @@ contract Attestations is
   address constant REQUEST_ATTESTATION = address(0xff);
 
   // The duration in seconds in which an attestation can be completed
-  uint256 public attestationExpirySeconds;
+  uint256 public attestationExpiryBlocks;
```

Comment:
Should we update the `migrationsConfig`? Presumably this value should be different now by a factor of 5